configure/recipes/default.rb

```diff
@@ -1,3 +1,14 @@
+node.default[:commit_hash] = "null"
+
+ruby_block "jbx_version" do
+  block do
+    %s(su #{node[:user]; git config --global --add safe.directory #{node["jbx"]["path"]})
+    commit_hash = %x(cd #{node["jbx"]["path"]}; git rev-parse HEAD)
+    node.default[:commit_hash] = "#{commit_hash.strip}"
+  end
+  action :run
+end
```

Comment:
can you localize this to the jbx cookbook, we shouldn't assume `jbx` is configured in all paths